openhtf/core/__init__.py

```diff
@@ -133,28 +140,14 @@ def _thread_proc(self):
         # Tear down plugs first, then output test record.
         exit_stack.callback(self.test_state.plug_manager.tear_down_plugs)
 
-        # Perform initialization of some top-level stuff we need.
-        executor = self._make_phase_executor(exit_stack)
-
       # Everything is set, set status and begin test execution.  Note we don't
       # protect this with a try: block because the PhaseExecutor handles any
       # exceptions from test code.  Any exceptions here are caused by the
       # framework, and we probably want them to interrupt framework state
       # changes (like the transition to FINISHING).
+      self.test_state.mark_test_started()
```

Comment:
I didn't quite realize the importance of some of the subtleties of moving this. I'll put it back up to right after the start trigger finishes.